templates/index.html

```diff
@@ -120,7 +117,7 @@ <h3 class="p-heading-icon__title">Vanilla Framework</h3>
           <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/9a5b9c51-sketch-icon.svg" alt="Sketch library:" />
           <h3 class="p-heading-icon__title">Vanilla Design</h3>
         </div>
-        <p>Get our Sketch library of common design components.</p>
+        <p>Get our Sketch library of common design components. The UI Kit is compatible with Vanilla v2.25.0</p>
```

Comment:
I think we need to make this more prominent. Maybe we could put it under Download link, instead of file size, with a warning icon, and say something like: "The Sketch UI Kit is no longer maintained. Current version is compatible with Vanilla v.2.25 from March 2021".